.travis.yml

```diff
@@ -10,10 +10,12 @@ compiler:
 env:
   global:
     - INSTALL_DIR=~/mylibs
+    - CMAKE_FLAGS="-DCMAKE_CXX_FLAGS=\"-Werror\" -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR"
```

Comment:
I had taken the idea from https://github.com/ComputationalRadiationPhysics/cupla/blob/dev/.travis.yml#L41, since the old way didn't work ( `CXX_FLAGS="-Werror" cmake` ), probably because the current value of CXX_FLAGS was not integrated into the Makefile (as I blindly assumed).
Storing it as a global env-var will probably work (fix incoming!)